CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [29.0.1]
+### Added
+- Add utils to format token list tokens for aggregators and icon url. ([#810](https://github.com/MetaMask/controllers/pull/810))
+- Add SupportedTokenDetectionNetworks enum and util for token detection support. ([#811](https://github.com/MetaMask/controllers/pull/811))
```

Comment:
Hmm.... usually we use a minor bump for releases for additions.

I guess we can let it slide this time, since this isn't meant to be an externally-facing change, and we're in a hurry. Unfortunate that all of our util functions are being exported right now, even those that are only used internally.

Reply:
yeah I was wondering about this. But also figured we'd decided to not revert these commits so